CMakeLists.txt

```diff
@@ -6,5 +6,12 @@ cmake_minimum_required(VERSION 3.5)
 # This example uses an extra component for common functions such as Wi-Fi and Ethernet connection.
 # set(EXTRA_COMPONENT_DIRS $ENV{IDF_PATH}/examples/common_components/protocol_examples_common)
 
+set(SDKCONFIG "${CMAKE_BINARY_DIR}/sdkconfig")
```

Comment:
Why move the SDKCONFIG path from the default?

Reply:
For whatever reason if I remove this line or point it to the root directory the sdkconfig.defaults don't work.